apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+(function (exports) {
+
+  var MoreAppsNavigation = {
+    controls: null,
+    selectedElemIndex: 0,
+    selectElement: null,
+
+    init: function () {
+      this.controls = app.grid.getItems();
+      this.reset();
+    },
+
+    reset: function () {
+      if (this.selectElement) {
+        this.selectElement.classList.remove('selected');
+      }
+      this.selectedElemIndex = 0;
+      this.selectElement = this.controls[this.selectedElemIndex].element;
+      this.selectElement.classList.add('selected');
+      this.clearNavigationEvents();
+      window.addEventListener('keydown', this);
+    },
+
+    clearNavigationEvents: function () {
+      window.removeEventListener('keydown', this);
+    },
+
+    handleEvent: function (e) {
+      var deltaIndex = 0;
+      e.preventDefault();
```

Comment:
don't call preventDefault until you know you're going to actually handle the keys. This line breaks the volume keys, for example.  You'll need to call preventDefault in each of the individual cases below.